static/src/javascripts/projects/common/modules/navigation/search.js

```diff
@@ -97,11 +125,16 @@ class Search {
                                                 'is-active'
                                             )
                                         ) {
+                                            popup.focus();
                                             bean.on(
                                                 document,
                                                 'click',
                                                 maybeDismissSearchPopup
                                             );
+                                            document.addEventListener(
+                                                'keyup',
+                                                handleEsc
+                                            );
```

Comment:
I wonder if we should unbind these `click` and `keyup` event handlers once `dismissSearchPopup` has been called?